packages/block-library/src/navigation/index.php

```diff
@@ -204,8 +205,16 @@ function core_block_navigation_build_html( $attributes, $block, $colors, $font_s
 
 	foreach ( (array) $block['innerBlocks'] as $key => $block ) {
 		$has_submenu = count( (array) $block['innerBlocks'] ) > 0;
+		$is_active   = ! empty( $block['attrs']['id'] ) && ( get_the_ID() === $block['attrs']['id'] );
+
+		$class_name = ! empty( $block['attrs']['className'] ) ? implode( ' ', (array) $block['attrs']['className'] ) : false;
+
+		if ( false !== $class_name ) {
+			$css_classes .= ' ' . $class_name;
+		};
 
-		$html .= '<li class="' . esc_attr( $css_classes . ( $has_submenu ? ' has-child' : '' ) ) . '"' . $style_attribute . '>' .
+		$html .= '<li class="' . esc_attr( $css_classes . ( $has_submenu ? ' has-child' : '' ) ) .
+			( $is_active ? ' current-menu-item' : '' ) . '"' . $style_attribute . '>' .
```

Comment:
To follow the pattern set by other class names, shouldn't it be something like `is-current-navigation-link`?

Reply:
I think the `is-` prefix fits pretty good here, in the same way that `is-selected`, etc... does.

Reply:
yes but all WP menus use this convention of current-menu-item, check [wp_nav_menu](https://developer.wordpress.org/reference/functions/wp_nav_menu/#menu-item-css-classes). There is a whole section of default CSS classes and this is-* convention isn't there.

<< I do agree >> with the is-* convention it is just unclear when we start breaking backwards compat.

Reply:
The changes made to the Navigation block won't have any effect on existing themes (which don't use blocks at all), so I see no problem with using an `is-` class here. But maybe someone else has another opinion?

Reply:
adding both of them is too much? So, we could guarantee the compatibility with the wp_nav_menu(), and at the same time maybe adding a super basic style for `.is-current-navigation-link`?

```es
.is-current-navigation-link {
    border: 1px solid red; // ;-)
}
```

Reply:
Also, it could be handled from the sidebar too, in a similar way that submenus icon does

<img src="https://user-images.githubusercontent.com/77539/74040361-6eadab80-4978-11ea-931e-1a5c7b92afca.png" width="300px" />

Reply:
@retrofox That sounds like overkill to me. It should be entirely up to the theme to determine if the current navigation link recieves special styles or not.

Reply:
Fair enough. Just bringing here the idea that I could be handled in both ways. Through the theme and/or a Navigation option.